ragna/deploy/_ui/central_view.py

```diff
@@ -405,148 +390,107 @@ def chat_interface(self):
 
         chat_interface = RagnaChatInterface(
             callback=self.chat_callback,
-            callback_user="Ragna",
+            user=self.user,
             show_rerun=False,
             show_undo=False,
             show_clear=False,
             show_button_name=False,
             view_latest=True,
             sizing_mode="stretch_width",
+            # TODO: @panel hitting enter to send a message is fine, but clicking
```

Comment:
I agree; we are eventually planning to have a custom Typescript chat text area input model that supports keyboard hotkeys and not send automatically when clicking away.

Reply:
An alternative is to use TextAreaInput, but the downside is you can't click enter to send the message.